### PR TITLE
Add Opensearch example

### PR DESCRIPTION
**Pull request by Galiley**

### CODE REVIEW

**Review by lpatte — DISMISSED** [review]

File: databases/opensearch/bash/hello-world/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Hello world for OpenSearch with bash
+
+The purpose of this tutorial is to execute a simple command on a OpenSearch database.
```
Comment (ybriffa) [inline databases/opensearch/bash/hello-world/README.md]: ... on **an** OpenSearch database.

File: databases/opensearch/bash/hello-world/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+## Requirements
+
+You need first to have a OpenSearch service available. [More information here](https://www.ovhcloud.com/en/public-cloud/opensearch/)
```
Comment (ybriffa) [inline databases/opensearch/bash/hello-world/README.md]: ...have **an** OpenSearch...

File: databases/opensearch/bash/hello-world/version.sh [hunk 3]
```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -e
+
+
+if [ $# -ne 4 ]
```
Comment (ybriffa) [inline databases/opensearch/bash/hello-world/version.sh]: can we take only the URI please ?

File: databases/opensearch/go/hello-world/README.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# Run a hello world on OpenSearch with go
+
+The purpose of this tutorial is to execute a simple command on the opensearch cluster.
+
+## Requirements
+
+You need first to have a OpenSearch cluster deployed. [More information here](https://www.ovhcloud.com/fr/public-cloud/opensearch/)
```
Comment (ybriffa) [inline databases/opensearch/go/hello-world/README.md]: ...to have **an** Opensearch cluster...

File: databases/opensearch/php/hello-world/README.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
+# Run a hello world on OpenSearch with php
+
+The purpose of this tutorial is to execute a simple command on the OpenSearch cluster.
+
+## Requirements
+
+You need first to have a OpenSearch cluster deployed. [More information here](https://www.ovhcloud.com/fr/public-cloud/opensearch/)
```
Comment (ybriffa) [inline databases/opensearch/php/hello-world/README.md]: an Opensearch

File: databases/opensearch/python/hello-world/README.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
+# Hello world for OpenSearch with python
+
+The purpose of this tutorial is to execute a simple command on a OpenSearch database.
+
+## Requirements
+
+You need first to have a OpenSearch service available. [More information here](https://www.ovhcloud.com/en/public-cloud/opensearch/)
```
Comment (ybriffa) [inline databases/opensearch/python/hello-world/README.md]: an Opensearch

File: databases/opensearch/python/hello-world/README.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+
+Once you have created your user, and set up your IP restrictions you can run the following:
+
+```console
+read USER
+read -s PASSWORD
+read URI
+docker build -t opensearch-python-hello-world .
+docker run opensearch-python-hello-world "$(echo -n $URI | sed "s/<username>/$USER/" | sed "s/<password>/$PASSWORD/")"
+```
```
Comment (ybriffa) [inline databases/opensearch/python/hello-world/README.md:19]: add the dependencies please :)

File: databases/opensearch/go/hello-world/README.md [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+
+Once you have created your user, and set up your IP restrictions you can run the following:
+
+```console
+read USER
+read -s PASSWORD
+read URI
+docker build -t opensearch-go-hello-world .
+docker run opensearch-go-hello-world "$(echo -n $URI | sed "s/<username>/$USER/" | sed "s/<password>/$PASSWORD/")"
+```
```
Comment (ybriffa) [inline databases/opensearch/go/hello-world/README.md]: add the dependencies please

**Review by ybriffa — APPROVED** [review]

**Review by julienisapunk — APPROVED** [review]

**Review by BellionBastien — APPROVED** [review]

**Pull request merged**